<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationWidget</class>
 <widget class="QWidget" name="AnimationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1392</width>
    <height>893</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Animation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/resource.qrc">
    <normaloff>:/16x16_ico_png/animation.png</normaloff>:/16x16_ico_png/animation.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Animation</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Base  properties</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="1">
              <widget class="QLineEdit" name="name">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="transform">
          <property name="title">
           <string>Transform</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>X pos</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="translateX">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimum">
                <double>-1000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000.000000000000000</double>
               </property>
               <property name="stepType">
                <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>X Scale</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3" rowspan="2">
              <widget class="QDoubleSpinBox" name="scaleX">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <double>1000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="stepType">
                <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="0" rowspan="2">
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>Y pos</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" rowspan="2">
              <widget class="QDoubleSpinBox" name="translateY">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimum">
                <double>-1000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000.000000000000000</double>
               </property>
               <property name="stepType">
                <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Y Scale</string>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QDoubleSpinBox" name="scaleY">
               <property name="maximum">
                <double>1000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="stepType">
                <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>Rotation</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="rotation">
               <property name="wrapping">
                <bool>true</bool>
               </property>
               <property name="minimum">
                <double>-180.000000000000000</double>
               </property>
               <property name="maximum">
                <double>180.000000000000000</double>
               </property>
               <property name="stepType">
                <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QPushButton" name="plus90">
               <property name="text">
                <string>+ 90°</string>
               </property>
              </widget>
             </item>
             <item row="3" column="3">
              <widget class="QPushButton" name="minus90">
               <property name="text">
                <string>- 90°</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="resetTransform">
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="gui::GfxWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Time</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="time">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Zoom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="zoom">
            <property name="minimum">
             <double>0.100000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="stepType">
             <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Components</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QListView" name="components">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="cProperties">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Visual properties</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Pass </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="materials">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Material</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>Layer</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="renderPass">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QSpinBox" name="layer">
               <property name="minimum">
                <number>-100</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="cName">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="cTransform">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Transform</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="1" column="0">
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>X pos</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="cTranslateX">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimum">
                <double>-10000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>10000.000000000000000</double>
               </property>
               <property name="stepType">
                <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QDoubleSpinBox" name="cRotation">
               <property name="wrapping">
                <bool>true</bool>
               </property>
               <property name="minimum">
                <double>-180.000000000000000</double>
               </property>
               <property name="maximum">
                <double>180.000000000000000</double>
               </property>
               <property name="stepType">
                <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="cTranslateY">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimum">
                <double>-10000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>10000.000000000000000</double>
               </property>
               <property name="stepType">
                <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QDoubleSpinBox" name="cSizeX">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimum">
                <double>-10000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>10000.000000000000000</double>
               </property>
               <property name="stepType">
                <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QPushButton" name="cPlus90">
               <property name="text">
                <string>+ 90°</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_26">
               <property name="text">
                <string>Y pos</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>Width</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_28">
               <property name="text">
                <string>Rotation</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Height</string>
               </property>
              </widget>
             </item>
             <item row="3" column="3">
              <widget class="QDoubleSpinBox" name="cSizeY">
               <property name="minimum">
                <double>-10000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>10000.000000000000000</double>
               </property>
               <property name="stepType">
                <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
               </property>
              </widget>
             </item>
             <item row="4" column="3">
              <widget class="QPushButton" name="cMinus90">
               <property name="text">
                <string>- 90°</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionNewRect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/32x32_ico_png/rectangle.png</normaloff>:/32x32_ico_png/rectangle.png</iconset>
   </property>
   <property name="text">
    <string>New Rect</string>
   </property>
  </action>
  <action name="actionDeleteComponent">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/16x16_ico_png/delete.png</normaloff>:/16x16_ico_png/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete animation component</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/32x32_ico_png/play.png</normaloff>:/32x32_ico_png/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/32x32_ico_png/stop.png</normaloff>:/32x32_ico_png/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/32x32_ico_png/pause.png</normaloff>:/32x32_ico_png/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionNewCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/32x32_ico_png/circle.png</normaloff>:/32x32_ico_png/circle.png</iconset>
   </property>
   <property name="text">
    <string>New Circle</string>
   </property>
   <property name="toolTip">
    <string>New Circle</string>
   </property>
  </action>
  <action name="actionNewTriangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/32x32_ico_png/triangle.png</normaloff>:/32x32_ico_png/triangle.png</iconset>
   </property>
   <property name="text">
    <string>New Triangle</string>
   </property>
   <property name="toolTip">
    <string>New Triangle</string>
   </property>
  </action>
  <action name="actionNewArrow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/32x32_ico_png/arrow.png</normaloff>:/32x32_ico_png/arrow.png</iconset>
   </property>
   <property name="text">
    <string>New Arrow</string>
   </property>
   <property name="toolTip">
    <string>New Arrow</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/32x32_ico_png/save.png</normaloff>:/32x32_ico_png/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save animation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>gui::GfxWidget</class>
   <extends>QWidget</extends>
   <header>gfxwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
